prowler/providers/aws/lib/quick_inventory/quick_inventory.py

```diff
@@ -244,10 +247,14 @@ def create_output(resources: list, audit_info: AWS_Audit_Info, output_directory:
     json_object = json.dumps(json_output, indent=4)
 
     # Writing to sample.json
-    with open(output_file + json_file_suffix, "w") as outfile:
+    with open(
+        args.output_directory + "/" + output_file + json_file_suffix, "w"
+    ) as outfile:
         outfile.write(json_object)
 
-    csv_file = open(output_file + csv_file_suffix, "w", newline="")
+    csv_file = open(
+        args.output_directory + "/" + output_file + csv_file_suffix, "w", newline=""
+    )
```

Comment:
Why the `json` is using the context manager and the `csv` don't?

Reply:
It was already like that